python/django/python-hello-world/kubernetes-manifests/hello.migration.yaml

```diff
@@ -0,0 +1,25 @@
+# This file creates a job that migrates the Postgres database. The job creates a
+# pod from the specified Docker image, wait for the database to get ready, and
+# run database migration once the Postgres database service is ready to receive
+# connections.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: python-hello-world-migration
+spec:
+  template:
+    metadata:
+      name: python-hello-world-migration
+    spec:
+      initContainers:
+      # Wait for the database to get ready before doing migration
+      - name: check-db-ready
+        image: postgres
+        command: ['sh', '-c',
```

Comment:
ditto this

Reply:
Removed database for hello-world template.